src/root/views/DrefApplicationForm/index.tsx

```diff
@@ -486,6 +496,36 @@ function DrefApplication(props: Props) {
 
   const failedToLoadDref = !pending && isDefined(drefId) && !drefResponse;
 
+  const handleDocumentImport = React.useCallback(async (newValue: File | undefined) => {
+    if (!newValue) {
+      return;
+    }
```

Comment:
Let's show some error.

Reply:
We don't need to, this is in case the value selected was not defined